### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by vboo123**
Still working on UI and navigation between pages

### CODE REVIEW

**Comment by ctperry0301** [comment]
Please respond to this comment when you are done with milestone 2

**Comment by vboo123** [comment]
I am done and submitted it saturday

**Comment by vboo123** [comment]
oh sorry, just saw the comment

**Review by ctperry0301 — COMMENTED** [review]
The code in your app.js and index.js (you should only need one of these) is just the code that exists on the demo and in the notion. That code is meant to be a resource to help you, not to be copy/pasted into your own code. Also your recipes.json was just taken from my example on milestone-2. It also looks as if your recipes.json looks like it was taken from my example project as well. 

If this code was submitted for a class, this would be blatant plagiarism and would probably have some pretty severe consequences, and I think you know that. Luckily, I'm not a teacher, and I just want all H4I developers to learn the skills needed to develop modern full stack web-applications. However, Your work in this milestone does not indicate that you have a proficient understanding of  javascript.  So I ask that you re-try this milestone again, using your **own** code and completing **all** of the requirements, as well as fixing all of the issues mentioned in my review for milestone-1. Once you do that, please make a new pull request and I will review your code again.

As always, feel free to reach out to me on slack if you have any questions, or if any of this doesn't make sense. Also, feel free to drop by our next office hours, as they are a great resource to get hands on help.

**Comment by vboo123** [comment]
I understand…I was somehow under the impression that we had to make it look
as exact as to ur demos. Will redo it tonight and re submit. Sorry about
that

Thanks

**Comment by vboo123** [comment]
Here is my second attempt. I had a couple questions with comments in the code itself. Also for my UI by the third milestone while implenting react I should have it looking "nice" as I am planning on using material UI, of course, if thats fine with you. Looking back at the instructions, I thought we had to completely copy your website, but I realize we can use the code in the demos in context with our own website. Also, when I try converting JSON object to JSON data, its return null for the values, not sure why!

**Comment by vboo123** [comment]
for the json issue, do I have to store my data in a seperate JSON file? Im looking back at it now and trying to really understand what is being asked

**Comment by ctperry0301** [comment]
> for the json issue, do I have to store my data in a seperate JSON file? Im looking back at it now and trying to really understand what is being asked

We can use a js feature called DOM Manipulation to access things like elements, text or html tags from our HTML files. So when a button on one of your car pages is clicked, we can use DOM Manipulation to modify a JSON object that represents the content that exists in our html page. Then, our goal for milestone-2 is to output that json on our page. Ill provide a code example in a few minutes to clarify things a little bit more.

**Comment by ctperry0301** [comment]
> > for the json issue, do I have to store my data in a seperate JSON file? Im looking back at it now and trying to really understand what is being asked
> 
> We can use a js feature called DOM Manipulation to access things like elements, text or html tags from our HTML files. So when a button on one of your car pages is clicked, we can use DOM Manipulation to modify a JSON object that represents the content that exists in our html page. Then, our goal for milestone-2 is to output that json on our page. Ill provide a code example in a few minutes to clarify things a little bit more.

Screen Shot 2021-11-02 at 6 34 21 PM

Does this answer your question?

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+<body>
+  <main>
+  <div class="intro-box">
+    <p>Welcome and thanks for stopping by my car blog! I will post some of my favorite cars and information regarding their specs!</p>
+    <img id="Home_Picture" src="https://hips.hearstapps.com/hmg-prod/amv-prod-cad-assets/images/16q3/669463/lightning-lap-2016-the-years-hottest-performance-cars-at-vir-feature-car-and-driver-photo-670701-s-original.jpg?resize=480:*" </img>
+  </div>
+  <div>
+    <h1>Cars</h1>
+    <h1>BMW M5</h1>
```
Comment (ctperry0301) [inline index.html:22]: Add links to your site so it's easier to navigate between pages.

```<a href="carOne.html"><h1>BMW M5</h1> </a> ``` should do the trick

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    <img id="Vehicle_Picture" src="https://cdn1.mecum.com/auctions/fl0120/fl0120-397316/images/1-1570457461398@2x.jpg?1576795854000" </img>
+    <h1>BMW M3</h1>
```
Comment (ctperry0301) [inline index.html:24]: Add a link here as well

**Review by ctperry0301 — COMMENTED** [review]
This site is a lot better! Great work!!! The final thing to do is just add the linking between the pages on your site, and to make sure that the json data gets outputted properly. Let me know if you need any help with this, it might be helpful for us to walk through it together if you're still confused. 

After those are fixed, you should be good to move on to milestone-3. Let me know if you have any questions and you have done a great job so far!! 👍

**Comment by vboo123** [comment]
Hi, sorry for the late response. I haven't gotten a chance to work on this because I have been really sick last one week. Poly plague probably. I'll finish these changes up by tomorrow and then work on milestone 3. Ill let you know if I have any questions. Thanks!

**Pull request closed without merging**